x-pack/legacy/plugins/rollup/public/crud_app/sections/job_create/steps/step_review.js

```diff
@@ -122,12 +127,25 @@ export class StepReviewUi extends Component {
         {this.renderTabs()}
 
         <EuiErrorBoundary>
-          <JobDetails
-            job={job}
-            json={json}
-            tab={selectedTab}
-          />
+          <JobDetails job={job} json={json} tab={selectedTab} />
         </EuiErrorBoundary>
+
+        <EuiSpacer size="m" />
+
+        <EuiCheckbox
+          id="rollupJobToggleJobStartAfterCreation"
+          data-test-subj="rollupJobToggleJobStartAfterCreation"
+          checked={startJobAfterCreation}
+          label={
+            <span>
+              <FormattedMessage
+                id="xpack.rollupJobs.create.startJobImmediately"
+                defaultMessage="Start job immediately"
```

Comment:
cc @gchaps any feedback on this wording?

Reply:
I'd recommend "Start job now"